modules/bastion/main.tf

```diff
@@ -72,7 +103,7 @@ resource "null_resource" "ipxe_files" {
 
     content = templatefile("${path.module}/assets/ipxe.tpl", {
       node_type           = each.value
-      bastion_ip          = equinix_metal_device.lb.access_public_ipv4
+      bastion_ip          = equinix_metal_device.lb.access_private_ipv4
```

Comment:
access_private_ipv4 needs to be replaced with an address from the VRF range, 192.168.100.2